celestia.vcxproj

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\windows\vcpkg\scripts\buildsystems\msbuild\vcpkg.props" />
```

Comment:
is it possible to use `vcpkg integrate install` to configure vcpkg integration so we don't need to add it as a submodule?